ui/src/app/i18n-helper.ts

```diff
@@ -0,0 +1,197 @@
+/**
+ * All strings used in the UI.
+ */
+export interface StringAssets {
+  submitButton: string;
+  consentMessageTitle: string;
+  consentMessageBodyTexts: string[];
+  languageSelect: string;
+  consentButton: string;
+}
+
+/**
+ * The list of language code which supported by G Suite. Google meet supports
+ * all of them.
+ * See more information at
+ * https://developers.google.com/admin-sdk/directory/v1/languages.
+ * NOTE: right to left languages are not supported yet so they are excluded.
+ */
+export enum LanguageCode {
+  Amharic = 'am',
+  // Arabic='ar',
+  Basque = 'eu',
+  Bengali = 'bn',
+  'English(UK)' = 'en-GB',
```

Comment:
Are these parenthesized values going to be used a lot? It seems like they'll make the enum somewhat annoying to work with. Are these intended to be user visible? Why not just something like EnglishUk?

Reply:
Agree that it is not easy to work with in the code. But I have two reasons which I think outweighs the cons:
1. When someone wants to add a new language, the person needs to look up the corresponding locale code from here. So it might be better to keep them more readable.
2. I also want to keep it the same as the source that I found. 

So I kept the format. Please let me know if it makes sense.